<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPerfectMatching</class>
 <widget class="QWidget" name="frmPerfectMatching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of Elements By Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSize">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>0000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdBuildModel">
       <property name="text">
        <string>Build Model</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tblManPref">
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <row/>
         <row/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblWomanPref">
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <row/>
         <row/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lytResult">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Run Algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblMatch">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <row/>
         <row/>
         <column/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtLog">
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cmdClear">
     <property name="text">
      <string>Clear Log</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
